src/Services/CRM/Activity/Result/CommunicationResult.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * This file is part of the bitrix24-php-sdk package.
+ *
+ * © Maksim Mesilov <mesilov.maxim@gmail.com>
+ *
+ * For the full copyright and license information, please view the MIT-LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Bitrix24\SDK\Services\CRM\Activity\Result;
+
+use Bitrix24\SDK\Services\CRM\Common\Result\AbstractCrmItem;
+
+/**
+ * @see https://apidocs.bitrix24.ru/api-reference/crm/timeline/activities/activity-base/crm-activity-communication-fields.html
+ *
+ * @property-read int $ID
+ * @property-read int $ACTIVITY_ID
+ * @property-read int $ENTITY_ID
+ * @property-read int $ENTITY_TYPE_ID
+ * @property-read string $TYPE
+ * @property-read string $VALUE
+ */
+class CommunicationResult extends AbstractCrmItem
```

Comment:
if this *item you must add Item prefix to class name.

please add new  issue «find and add tool to check naming conversation», responsible - mesilov

Reply:
see https://github.com/bitrix24/b24phpsdk/blob/dev/docs/EN/Development/how-to-contribute.md find *Item